src/pages/Appearance/Programs/QuickActionsSettings.js

```diff
@@ -0,0 +1,60 @@
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
+import { CheckboxField } from '../../../components/field'
+import { TableRowWrapper } from '../../../components/table'
+
+const quickActionsList = [
+    { key: 'MARK_FOLLOW_UP', label: i18n.t('Mark for Follow Up') },
+    { key: 'TRANSFER', label: i18n.t('Transfer') },
+    { key: 'COMPLETE_ENROLLMENT', label: i18n.t('Complete Enrollment') },
+    { key: 'DEACTIVATE', label: i18n.t('Deactivate') },
+    { key: 'MORE_ENROLLMENTS', label: i18n.t('More Enrollments') },
+]
+
+export const QuickActionsSettings = ({ handleChange, settings }) => {
+    const [selectedActions, setSelectedActions] = useState([])
+    useEffect(() => {
+        if (settings?.quickActions) {
+            setSelectedActions(
+                settings.quickActions.map((action) => action.actionId)
+            )
+        }
+    }, [settings])
```

Comment:
Do we really need an `useEffect` here, wouldnt just setting the initialValue of `selectedActions` be enough? 
Will `settings.quickActions` ever update while this is rendered - and do we want that?